_data/communication.yml

```diff
@@ -1,4 +1,11 @@
 websites:
+    - name: Action Network
+      url: https://actionnetwork.org/
+      img: actionnetwork.png
+      tfa: Yes
+      software: Yes
+      doc: https://help.actionnetwork.org/hc/en-us/articles/217206826
```

Comment:
Sure, yes. We use Instant2FA for our authentication service. (https://instant2fa.com/)

By attach a screenshot, you mean here in this PR? Or in our help documentation?